Commands.adoc

```diff
@@ -0,0 +1,226 @@
+= Available Commands and Examples
+:toc:
+:toc-placement!:
+
+Once you do `vagrant up` and run an ADB box, as a developer you may want to
+customize or set up your development environment to optimally utilize the ADB as
+ per your requirements.
```

Comment:
Can we not have simpler sentences?

> Once you `vagrant up` and have a running VM,  vagrant-service-manager enables you to check and tweak your container development environment.

Says imo opinion the same thing and easier to understand